apps/studio/src/utils/link.ts

```diff
@@ -0,0 +1,21 @@
+interface GetReferenceLinkParams {
+  siteId: string
+  resourceId: string
+}
+
+export const getReferenceLink = ({
+  siteId,
+  resourceId,
+}: GetReferenceLinkParams): string => {
+  return `[resource:${siteId}:${resourceId}]`
```

Comment:
should we validate this too? feels like this is injectable

Reply:
Added a regex test for numeric values in 7f139851126b5ab4f72d99fbbf9c5588cb2f9adc.